packages/host/src/components/AppBar/SectionIcon.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import {
+  ListIcon,
+  LocaleKey,
+  matchPath,
+  ReportsIcon,
+  RouteBuilder,
+  StockIcon,
+  SuppliersIcon,
+  Tooltip,
+  TruckIcon,
+  useLocation,
+  useTranslation,
+} from '@openmsupply-client/common';
+import { AppRoute } from '@openmsupply-client/config';
+
+const getSection = () => {
+  const routes = [
+    AppRoute.Catalogue,
+    AppRoute.Distribution,
+    AppRoute.Inventory,
+    AppRoute.Replenishment,
+    AppRoute.Reports,
+  ];
+  const location = useLocation();
+
+  for (let i = 0; i < routes.length; i++) {
+    const route = routes[i];
+    const match = matchPath(
+      RouteBuilder.create(route ?? '')
+        .addWildCard()
+        .build(),
+      location.pathname
+    );
+    if (!!match) return route;
+  }
+  return undefined;
+};
+
+export const SectionIcon: React.FC = () => {
+  const t = useTranslation('app');
+  const section = getSection();
+  const icon = getIcon(section);
+
+  return icon ? (
+    <Tooltip title={t(section as LocaleKey)}>
```

Comment:
iffy, and probably not required? not sure that we need a tooltip on the icon.. generally think that it is helpful tho